CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# How can you make your first **Pull Request**
+# How can I contribute?
 
```

Comment:
Please check the same section on website-my, you might like it 🙂 
https://github.com/Real-Dev-Squad/website-my/blob/develop/CONTRIBUTING.md#how-can-i-contribute

Reply:
https://github.com/Real-Dev-Squad/website-members/pull/145/commits/19474dd80b39b9708d6b67626e1cc8adabcb1021